server/graphql/v1/mutations/orders.js

```diff
@@ -867,9 +867,17 @@ export async function markOrderAsPaid(remoteUser, id) {
   if (order.status !== 'PENDING') {
     throw new ValidationFailed("The order's status must be PENDING");
   }
-  const HostCollectiveId = await models.Collective.getHostCollectiveId(order.CollectiveId);
-  if (!remoteUser.isAdmin(HostCollectiveId)) {
-    throw new Unauthorized('You must be logged in as an admin of the host of the collective');
+
+  const collective = await models.Collective.findByPk(order.CollectiveId);
+  if (collective.isHostAccount) {
+    if (!remoteUser.isAdmin(collective.id)) {
```

Comment:
:ice_cream: You can simplify scopes by combining these conditionals.

Reply:
If I just combine them, I'll need to update the `else` to an `else if` or to create a new `if` block because we don't want it to run if the collective is not a host (ie. `if (isHost && !isAdmin) { ... } else if (!isHost) { ... }`)

Reply:
Oh, I get it! Thanks!